lib/run-script.js

```diff
@@ -84,31 +84,52 @@ function runScript (args, cb) {
 
 function list(cb) {
   var json = path.join(npm.localPrefix, "package.json")
+  var cmdList = [ "publish", "install", "uninstall"
+                , "test", "stop", "start", "restart"
+                ].reduce(function (l, p) {
+                  return l.concat(["pre" + p, p, "post" + p])
+                }, [])
   return readJson(json, function(er, d) {
     if (er && er.code !== "ENOENT" && er.code !== "ENOTDIR") return cb(er)
     if (er) d = {}
-    var scripts = Object.keys(d.scripts || {})
+    var allScripts = Object.keys(d.scripts || {})
+    var scripts = [], runScripts = []
```

Comment:
We're moving npm to follow @feross's `standard` code style, which disallows multiple assignment. Split this into two declarations, please?